client_plugin/Makefile

```diff
@@ -424,19 +424,19 @@ test-e2e-shared:
 .PHONY: test-e2e-runalways test-e2e-runonce test-e2e-runonce-windows
 test-e2e-runalways:
 	$(log_target)
-	$(GO) test -v -timeout 40m -tags "runalways" $(E2E_Tests)
+	$(GO) test -v -timeout 50m -tags "runalways" $(E2E_Tests)
```

Comment:
Since all of the commands are triggered through esxcli, its adding a bit more time to overall time of running tests (since we fire so much admin commands over ssh)